### ISSUE TITLE
Manual Install awesome startup error

### ISSUE THREAD

**Issue by Dadams2** [szorfein/dots#2]
Followed the manual install instructions on a fresh arch install and got the following errors when running ```startx```

Oops, there were errors during startup!
error while running function
stack trackback:
    ./module/menu.lua:19: in main chunk
    [C]: in function 'require'
   /home/ninja.config/awesome/rc.lua:38: in main chunk
error: .module/menu.lua:19: attempt to concatenate a nil value (global 'terminal_cmd')

I am unsure of how to fix this problem or what other information is needed for diagnosis

**Comment by Dadams2** [comment]
Chezmoi apply also didn't do anything on gentoo fresh install due to cloning into /root/.local/share/chezmoi

**Comment by szorfein** [comment]
I just finished the installation on Archlinux and no problem, I have add unzip to the dependencies to install the font Cyberpunk.  
You have install this on root? try the install with a normal user.

For Gentoo, it will take more time to test :+1: 

**EDIT** I've correct the [wiki](https://github.com/szorfein/dotfiles/wiki/theme-awesome) for the manual install, you need the config file.

**Comment by Dadams2** [comment]
Have tried again following each step in order. I got the following error:

```
archtest% stow lines -t ~
WARNING! stowing lines would cause conflicts:
   * existing target is now owned by stow: .config/awesome
all operations aborted
```
i believe this is occurring as .config/awesome has two different places where stow is trying to symlink to. I personally use git bare repositories to store dotfiles so am not entirely sure what the issue with stow. 

I had to manually copy all files from the themes/lines directory into ~/ which did work. Thanks so much for the awesome themes!
